components/vacay.css

```diff
@@ -40,6 +42,7 @@
     --yellow-color-200:#FDE68A;
     --yellow-color-900:#78350F;
     --red-color-200:#FECACA;
+    --red-color-400:#EF4444;
```

Comment:
just a suggestion. you can change naming to capital letters like --RED-COLOR-400 to increase readability. Rest, nice work! 👍

Reply:
Is it a good practice @nayyyhaa ? I saw in MDN they were using lowercase.